File: jat-start-up-core/src/main/java/com/epam/jatstartup/dto/MeetingSimpleInfoDTO.java
package com.epam.jatstartup.dto;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDateTime;
import java.util.List;

@Data
@Builder
public class MeetingSimpleInfoDTO {

    private String type;
    private int thread;
    private LocalDateTime start;
    private LocalDateTime end;
    private ParticipantDTO interviewee;
    private List<ChosenTopicThemeDTO> subjects;

}

File: jat-start-up-ui/src/models/MeetingSimpleInfoDTO.ts
import { ChosenTopicThemeDTO } from './ChosenTopicThemeDTO';
import { ParticipantDTO } from './ParticipantDTO';

export interface MeetingSimpleInfoDTO {
    type: string;
    thread: number;
    start: Date;
    end: Date;
    interviewee: ParticipantDTO;
    subjects: ChosenTopicThemeDTO[];
}